lambdas/answer/src/main/java/uk/gov/di/ipv/cri/kbv/api/handler/QuestionAnswerHandler.java

```diff
@@ -71,22 +79,27 @@ public APIGatewayProxyResponseEvent handleRequest(
         try {
             processAnswerResponse(input);
         } catch (JsonProcessingException jsonProcessingException) {
-            LOGGER.error("Failed to parse object using ObjectMapper");
+            eventProbe.log(INFO, jsonProcessingException).counterMetric(POST_ANSWER, 0d);
```

Comment:
Should this be `eventProbe.log(INFO` or `Level.ERROR` seem for all the others where an occur has occurred but you've choosen INFO instead of ERROR

Reply:
I think this is fine since it something wrong with the request